plugins.json

```diff
@@ -3075,6 +3075,13 @@
         "date": "2025-05-16T21:11:05.374888809+02:00",
         "sha512sum": "cdf832090c9449a260578d068bd1b17d86e80b12f066953f7af49268d364b58f5cb88742cb722e403acb7f61f991aeff90f12a021214dd503322757686b20438",
         "requires": ">=24.04.0"
+      },
+      {
+        "version": "2.0.1",
+        "date": "2025-05-28T22:08:56.241806203+02:00",
+        "url": "https://github.com/CommonWorkflowScheduler/nf-cws/releases/download/2.0.1/nf-cws-2.0.1.zip",
+        "requires": ">=24.04.0",
```

Comment:
Did you mean to say `25.04` here? If you added support for 25.04 then I would expect it to not work with previous verisons

Reply:
Thanks for checking, but 24.04.0 is correct. Until now, the plugin worked only for >= 24.04.0 and < 25.03, as it missed the Kubernetes classes for newer releases. Including the classes is, luckily, backward compatible.

Reply:
I have doubts that it will work as well as you expect. Including both Nextflow 24.04 and nf-k8s means you'll have two copies of the K8s integration

I assume one will override the other, and I suspect they are near identical anyway since there has been little development on K8s over the last year. But if there are differences, it might lead to weird bugs

Have you considered this possibility?

Reply:
Yes, but the k8s classes are now added at build time, and I overwrite the default Kubernetes Executor in any case.
I tested with 24.04 and found no issues.
It is even better now, as all Kubernetes classes are now imported, so I no longer depend on files specific to a particular Nextflow version. The files are always the same.